services/QuillLMS/app/models/activity.rb

```diff
@@ -116,12 +116,18 @@ class Activity < ApplicationRecord
   # the other two, Proofreader and Lesson, contain passages and other data, not questions
   ACTIVITY_TYPES_WITH_QUESTIONS = [2,4,5]
 
-  STARTER_DIAGNOSTIC_ACTIVITY_ID = 1663
-  INTERMEDIATE_DIAGNOSTIC_ACTIVITY_ID = 1668
-  ADVANCED_DIAGNOSTIC_ACTIVITY_ID = 1678
-  ELL_STARTER_DIAGNOSTIC_ACTIVITY_ID = 1161
-  ELL_INTERMEDIATE_DIAGNOSTIC_ACTIVITY_ID = 1568
-  ELL_ADVANCED_DIAGNOSTIC_ACTIVITY_ID = 1590
+  STARTER_DIAGNOSTIC_ACTIVITY_ID = 2537
+  INTERMEDIATE_DIAGNOSTIC_ACTIVITY_ID = 2539
+  ADVANCED_DIAGNOSTIC_ACTIVITY_ID = 2541
+  ELL_STARTER_DIAGNOSTIC_ACTIVITY_ID = 2550
+  ELL_INTERMEDIATE_DIAGNOSTIC_ACTIVITY_ID = 2555
+  ELL_ADVANCED_DIAGNOSTIC_ACTIVITY_ID = 2563
+  OLD_STARTER_DIAGNOSTIC_ACTIVITY_ID = 1663
+  OLD_INTERMEDIATE_DIAGNOSTIC_ACTIVITY_ID = 1668
+  OLD_ADVANCED_DIAGNOSTIC_ACTIVITY_ID = 1678
+  OLD_ELL_STARTER_DIAGNOSTIC_ACTIVITY_ID = 1161
+  OLD_ELL_INTERMEDIATE_DIAGNOSTIC_ACTIVITY_ID = 1568
+  OLD_ELL_ADVANCED_DIAGNOSTIC_ACTIVITY_ID = 1590
   PRE_TEST_DIAGNOSTIC_IDS = [STARTER_DIAGNOSTIC_ACTIVITY_ID, INTERMEDIATE_DIAGNOSTIC_ACTIVITY_ID, ADVANCED_DIAGNOSTIC_ACTIVITY_ID, ELL_STARTER_DIAGNOSTIC_ACTIVITY_ID, ELL_INTERMEDIATE_DIAGNOSTIC_ACTIVITY_ID, ELL_ADVANCED_DIAGNOSTIC_ACTIVITY_ID]
```

Comment:
It might be worth checking to see if we need to include the old pre tests in here as well (I can't remember offhand where this gets used).